src/styles/variables.css

```diff
@@ -56,6 +56,50 @@
   /* font */
   --gotham: 'Gotham', Arial, Calibri, sans-serif;
 
+  /* font size */
+
+  --font-size: 1rem;
+  --line-height: 1.5rem;
+
+  --display1-font-size: 3.75rem;
+  --display1-line-height: 5.625rem;
```

Comment:
Let's use the Semantic nomenclature.

"subheading" = "h6"
"display1" = "huge"
"display2" = "large"
"p1" = "medium"
"p2" = "small"
"p3" = "tiny"